stylesheets/_typography.scss

```diff
@@ -40,13 +40,13 @@ $is-print: false !default;
   font-weight: $font-weight;
```

Comment:
Looks like we'd still inherit the line-height from here?

Only conditionally having the media query feels arbitrary, maybe it'd be clearer to override the line-height for print?